### PR TITLE
chore(project): minor style improvements

### PR DESCRIPTION
**Pull request by yezz123**
Most refactoring cases were Replacing conditional assignment to a variable with an if expression.

### CODE REVIEW

**Comment by RobertCraigie** [comment]
Before I do a full review, is there any underlying reason behind one-lining if statements or is it personal style preference?

File: tests/utils.py [hunk 1]
```diff
@@ -269,11 +269,7 @@ def assert_similar_time(dt1: datetime, dt2: datetime, threshold: float = 0.5) ->
 
     This issue does not appear to be solvable by us, please create an issue if you know of a solution.
     """
-    if dt1 > dt2:
-        delta = dt1 - dt2
-    else:
-        delta = dt2 - dt1
-
+    delta = dt1 - dt2 if dt1 > dt2 else dt2 - dt1
```
Comment (RobertCraigie) [inline tests/utils.py:272]: I find this more difficult to read, please revert this back to the original.

File: src/prisma/generator/models.py [hunk 2]
```diff
@@ -586,9 +586,8 @@ def scalar_type_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         kind = values.get('kind')
         type_ = values.get('type')
 
-        if kind == 'scalar':
-            if type_ is not None and type_ not in TYPE_MAPPING:
-                raise ValueError(f'Unsupported scalar field type: {type_}')
+        if kind == 'scalar' and type_ is not None and type_ not in TYPE_MAPPING:
+            raise ValueError(f'Unsupported scalar field type: {type_}')
```
Comment (RobertCraigie) [inline src/prisma/generator/models.py:590]: I prefer the original style as it makes it easier to add more cases in the future.

For example, if we needed to add another check for scalar types then with the old style we would just have to add a new nested if statement, however with this new style we will either have to duplicate the `kind == 'scalar'` check or extract it to it's own if statement, bringing the code back to its original state.

File: src/prisma/generator/errors.py [hunk 3]
```diff
@@ -9,11 +9,7 @@ class TemplateError(PrismaError):
 
 class CompoundConstraintError(ValueError):
     def __init__(self, constraint: Union['PrimaryKey', 'UniqueIndex']) -> None:
-        if isinstance(constraint, PrimaryKey):
-            annotation = '@@id'
-        else:
-            annotation = '@@unique'
-
+        annotation = '@@id' if isinstance(constraint, PrimaryKey) else '@@unique'
```
Comment (RobertCraigie) [inline src/prisma/generator/errors.py:12]: See comment on `src/prisma/generator/models.py`.

File: src/prisma/mypy.py [hunk 4]
```diff
@@ -155,11 +155,9 @@ def _handle_include(self, ctx: MethodContext) -> Type:
...
                 # TODO: add more details
                 # e.g. "include" to "find_unique" of "UserActions"
-                if isinstance(exc, UnparsedExpression):
-                    err_ctx = exc.context  # pylint: disable=no-member
-                else:
-                    err_ctx = include_expr
-
+                err_ctx = (
+                    exc.context if isinstance(exc, UnparsedExpression) else include_expr
+                )  # pylint: disable=no-member
```
Comment (RobertCraigie) [inline src/prisma/mypy.py:160]: See comment on src/prisma/generator/models.py.

It also localises our lint comment which I prefer.

**Review by RobertCraigie — CHANGES_REQUESTED** [review]
Thanks for the contribution!

**Comment by yezz123** [comment]
Hello @RobertCraigie, Sorry I miss checking the notification in GitHub and PRs that I Open I Just checked this while I was having a lot of works during these days, If we can enhance anything I can help!

Also I'm, working in a PR to Add Prisma-Client to AuthX

**Comment by RobertCraigie** [comment]
@yezz123 no worries! 

> Also I'm, working in a PR to Add Prisma-Client to AuthX

I'm more than happy to help! Let me know if you have any questions and I'm happy to look over the code when you're finished :)

**Comment by yezz123** [comment]
> @yezz123 no worries!
> 
> > Also I'm, working in a PR to Add Prisma-Client to AuthX
> 
> I'm more than happy to help! Let me know if you have any questions and I'm happy to look over the code :)

Yes, Lately in FastAPI server a developer asked about adding Prisma-Client Support and I see its time to do it, while i just drop out MongoDB and Add SQL Databases, i Will add Prisma also

**Comment by RobertCraigie** [comment]
Exciting! Prisma also supports MongoDB.

**Review by RobertCraigie — APPROVED** [review]

**Comment by RobertCraigie** [comment]
Thank you!

**Pull request merged**